web/src/components/tasks/business-formation/review/ReviewSignatures.tsx

```diff
@@ -24,10 +24,10 @@ export const ReviewSignatures = (): ReactElement => {
   return (
     <>
       <ReviewSectionHeader header={getHeader()} stepName="Contacts" testId="signature" />
-      {state.formationFormData.signers.length === 0 && (
+      {state.formationFormData.signers?.length === 0 && (
         <i>{Config.businessFormationDefaults.reviewStepNotEnteredText}</i>
       )}
-      {state.formationFormData.signers.map((signer, index) => {
+      {state.formationFormData.signers?.map((signer, index) => {
```

Comment:
why are we changing so many fields to be able to be undefined? especially a list - why not just empty list?